README.md

```
@@ -1,5 +1,41 @@
+    // Tratamento de erro
+}
+
+var users []User
+if err := resp.Body().AsJSON(&users); err != nil {
+    // Erro ao ler a resposta
+}
+```
+
+Outros exemplos estão disponíveis na pasta `examples`, incluindo chamadas com cabeçalhos customizados, balanceamento de carga e *tracing* com OpenTelemetry. E não se surpreenda se surgir uma nova referência à Mai no meio dos logs.
+
+## Contributing
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)